components/icons/RocketIcon.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2009-2020 SonarSource SA
+ * All rights reserved
+ * mailto:info AT sonarsource DOT com
+ */
```

Comment:
This will fail the license check, I gather 🙂 . If not... we should update license check.

Reply:
Arf, I always do that when moving things from SC repo to SUC...